### DISCUSSION TITLE
Disable the ability to drag and drop and connect elements

### DISCUSSION THREAD

**Discussion by QuangLanh** [clientIO/joint#2457] [Q&A]
### Introduction

I want to set a state for these elements, where users can only click on them, and they are not allowed to drag, drop, or link. This means I want to disable the drag-and-drop and link creation features when I set status for these elements. How can I do that?

### Steps to reproduce

![Screenshot 2023-12-29 231550](https://github.com/clientIO/joint/assets/91791283/5a69d6c0-f971-4d35-a7ba-dc9ad6cbbc1e)

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
Checkout the following demos that show how to adjust the interactivity: https://www.jointjs.com/demos?feature=Adjustable+interactivity

The demos use the [interactive](https://resources.jointjs.com/docs/jointjs/v3.7/joint.html#dia.Paper.prototype.options.interactive) paper option or the [elementView.preventDefaultInteraction()](https://resources.jointjs.com/docs/jointjs/v3.7/joint.html#dia.CellView.prototype.preventDefaultInteraction) method to prevent the default JointJS action.

**Comment by QuangLanh** [comment]
I have disabled the label (element) but I still cannot disable the link (connection).

**Reply by kumilingus** [reply]
And how do you create the links?
